vmware-event-router/hack/create_k8s_config.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash -eu
```

Comment:
Can we use set -euo pipefail (https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/) please?

Reply:
Done